src/style.css

```diff
@@ -1,3 +1,72 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
+  text-align: left;
+
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  padding-left: 50px;
+  padding-right: 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.logo {
+  align-content: center;
+  text-decoration: none;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style-type: none;
+  position: relative;
+  margin-right: 20px;
+}
+
+.nav__link {
+  align-content: center;
+  display: block;
+  color: black;
+  text-transform: uppercase;
+  font-size: 12px;
+  height: 60px;
```

Comment:
Vertical margins and heights are set on the `.nav__link` class. Be careful with setting fixed heights on elements as it can lead to issues with content overflow or clipping. It's better to use padding and allow the content to dictate the size of the element.